### ISSUE TITLE
Print out StableHLO representation for batch norm and dropout

### ISSUE THREAD

**Issue by vladimirjovanovicTT** [tenstorrent/tt-blacksmith#11]
We would like to showcase differences for forward passes in training and inference modes. For this, we would like to be able to see exact differences in StableHLO representations of two simplest networks:
- one with only a batch normalization layer
- one with only a dropout layer

You can copy/paste them into this issue.
